src/main/python/hydra/lib/h_analyser.py

```diff
@@ -0,0 +1,57 @@
+__author__ = 'AbdullahS'
+
+from pprint import pprint, pformat   # NOQA
+import zmq
+import time
+import logging
+import json
+from hydra.lib import util
+
+l = util.createlogger('HDaemon', logging.INFO)
+# l.setLevel(logging.DEBUG)
+
+
+class HAnalyserBase(object):
+    def __init__(self, **kwargs):
+        self.server_ip = kwargs.pop("server_ip")
+        self.port = kwargs.pop("server_port")
+        self.data = {}  # This is where all received data will be stored
+
+        self.context = zmq.Context()
+        l.info("Connecting to server at [%s:%s]", self.server_ip, self.port)
+        self.socket = self.context.socket(zmq.REQ)
+        self.socket.connect("tcp://%s:%s" % (self.server_ip, self.port))
+        l.info("Conneced...")
+
+class HAnalyser(HAnalyserBase):
+    def __init__(self, **kwargs):
```

Comment:
I can understand the use of kwargs if it's a pass through to another class here.